docs/launch-behavior.md

```diff
@@ -0,0 +1,165 @@
+| `atom --add /b/3.md`        | [] [/a,/b 3.md]   | [] [/a 3.md] :warning:    | [] [/a 3.md] :new:    |
+| `atom --add /b`             | [] [/a,/b]        | [] [/a,/b]                | [] [/a,/b]            |
+| `atom --new-window /a/1.md` | [] [/a] [/a 1.md] | [] [/a] [ 1.md] :warning: | [] [/a] [ 1.md] :new: |
+| `atom --new-window /a`      | [] [/a] [/a]      | [] [/a] [/a]              | [] [/a] [/a]          |
+| `atom --new-window /b/3.md` | [] [/a] [/b 3.md] | [] [/a] [ 3.md] :warning: | [] [/a] [ 3.md] :new: |
+| `atom --new-window /b`      | [] [/a] [/b]      | [] [/a] [/b]              | [] [/a] [/b]          |
+
+## File manager integration actions
+
+### No open windows
+
+When no Atom windows are open, file manager context operations have the following outcomes:
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md] | [ 1.md] :warning: | [ 1.md] :new: |
+| Click on `/a`      | [/a]      | [/a]              | [/a]          |
+
+### Open window, no project roots
+
+With the following starting state:
+
+> []
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md] | [] [ 1.md] :warning: | [ 1.md] :bug: :new: |
+| Click on `/a`      | [/a]      | [] [/a] :warning:    | [/a] :bug: :new:    |
+
+### Open window, project root
+
+With the following starting state:
+
+> [/a]
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md]      | [/a 1.md]              | [/a 1.md]       |
+| Click on `/a`      | [/a]           | [/a]                   | [/a]            |
+| Click on `/b/3.md` | [/a 3.md]      | [/a] [ 3.md] :warning: | [/a 3.md] :bug: |
+| Click on `/b`      | [/a] [/b]      | [/a] [/b]              | [/a] [/b]       |
+
+### Open windows, one with a project root and one without
+
+> [/a] []
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [/a 1.md] []      | [/a 1.md] []              | [/a 1.md] []             |
+| Click on `/a`      | [/a] []           | [/a] []                   | [/a] []                  |
+| Click on `/b/3.md` | [/a] [/b 3.md]    | [/a] [] [ 3.md] :warning: | [/a] [ 3.md] :bug: :new: |
+| Click on `/b`      | [/a] [/b]         | [/a] [] [/b] :warning:    | [/a] [/b] :bug:          |
+
+> [] [/a]
+
+| Action | Outcome (<=1.35.1) | Outcome (1.36.0) | Outcome (1.36.1) |
+| --- | --- | --- | --- |
+| Click on `/a/1.md` | [] [/a 1.md] | [] [/a 1.md]              | [] [/a 1.md]             |
+| Click on `/a`      | [] [/a]      | [] [/a]                   | [] [/a]                  |
+| Click on `/b/3.md` | [] [/a 3.md] | [] [/a] [ 3.md] :warning: | [ 3.md] [/a] :bug: :new: |
+| Click on `/b`      | [] [/a] [/b] | [] [/a] [/b]              | [/b] [/a] :bug: :new:    |
```

Comment:
This change feels like a "nice-to-have" to me. If we can make it work easily, then I think it's a nice improvement. Otherwise, I recommend that we defer it for a future pull request, since this change wasn't an explicit goal of #18608.